content/docs/reference/api-docs.md

```diff
@@ -1,10 +1,10 @@
 ---
 title: API Reference
 description: >-
-  Learn about the cert-manager API which includes Custom Resources such as
-  Certificate, CertificateRequest, Issuer and ClusterIssuer.
+  cert-manager API documentation, including Custom Resources such as
+  Certificate, CertificateRequest, Issuer and ClusterIssuer
 ---
-Learn about the cert-manager API which includes Custom Resources such as Certificate, CertificateRequest, Issuer and ClusterIssuer.
+<p>cert-manager API documentation, including various Custom Resource Definitions</p>
```

Comment:
note: the updated description is a stylistic choice. I think it's an improvement as we don't need to tell people they're gonna "learn" when they're on API reference docs. Learning is the point!

The `<p>` for the page heading is required to ensure that that text is always in a container.